src/common.h

```diff
@@ -92,18 +92,12 @@ void *strophe_realloc(const xmpp_ctx_t *ctx, void *p, size_t size);
 char *strophe_strdup(const xmpp_ctx_t *ctx, const char *s);
 char *strophe_strndup(const xmpp_ctx_t *ctx, const char *s, size_t len);
 
-void xmpp_log(const xmpp_ctx_t *ctx,
-              const xmpp_log_level_t level,
-              const char *area,
-              const char *fmt,
-              va_list ap);
-
 /* wrappers for xmpp_log at specific levels */
-void xmpp_error(const xmpp_ctx_t *ctx, const char *area, const char *fmt, ...);
-void xmpp_warn(const xmpp_ctx_t *ctx, const char *area, const char *fmt, ...);
-void xmpp_info(const xmpp_ctx_t *ctx, const char *area, const char *fmt, ...);
-void xmpp_debug(const xmpp_ctx_t *ctx, const char *area, const char *fmt, ...);
-void xmpp_debug_verbose(
+void log_error(const xmpp_ctx_t *ctx, const char *area, const char *fmt, ...);
```

Comment:
we use the same function names in Profanity btw

Reply:
:unamused: OK, I'll also put them in the `strophe_` space as otherwise static linking would fail ...